proposals/secure-storage.md

```diff
@@ -38,35 +38,82 @@ Response:
 ```
 {
   type: "MACOS_KEYCHAIN",
-  availableAuthentication: [
-    "PIN",
-    "PASSWORD",
-    "BIOMETRY_FACE",
-    "BIOMETRY_FINGERPRINT"
+  supportedAuthenticationLevels: [
+    {
+      name: "BIOMETRIC",
+      implementation: [
+        "MACOS_KEYCHAIN_FACE",
+        "MACOS_KEYCHAIN_TOUCHID"
```

Comment:
This would apply to iOS as well. I would prefer to not bake the platform into the contents like this. I much preferred the previous generic wording.